packages/core/src/domain/tracekit/computeStackTrace.spec.ts

```diff
@@ -499,6 +499,53 @@ Error: foo
     })
   })
 
+  it('should ignore custom error messages on chrome ', () => {
+    const stackFrames = computeStackTrace(new Error('bar@http://path/to/file.js:1:1'))
+
+    expect(stackFrames.stack[0]).not.toEqual({
+      args: [],
+      column: 1,
+      func: 'Error: bar',
+      line: 1,
+      url: 'http://path/to/file.js',
+    })
+  })
+
+  it('should ignore messages that come from the `toString` of the error on chrome ', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this test case is about the same as the test case above. Maybe remove it?